lib/std/Target/x86.zig

```diff
@@ -3081,6 +3081,11 @@ pub const cpu = struct {
             .xsaveopt,
         }),
     };
+    pub const @"i86": CpuModel = .{
+        .name = "i86",
+        .llvm_name = null,
+        .features = featureSet(&[_]Feature{}),
+    };
```

Comment:
Oops, I didn't catch this earlier. The way to add new CPUs to these files is to modify `tools/update_cpu_features.zig`. If you don't do this, this entry will be lost on the next LLVM update.